eox_tenant/tenant_wise/proxies.py

```diff
@@ -126,43 +158,98 @@ def get_value_for_org(cls, org, val_name, default=None):
         if cached_value:
             return cached_value
 
-        tenant_config = TenantConfig.objects.values_list("lms_configs")
-        microsite_config = Microsite.objects.values_list("values")  # pylint: disable=no-member
+        result = TenantConfig.get_value_for_org(org, val_name)
+
+        if result:
+            cls.set_key_to_cache(cache_key, result)
+            return result
+
+        result = Microsite.get_value_for_org(org, val_name)
+        result = result if result else default
+        cls.set_key_to_cache(cache_key, result)
+
+        return result
+
+    @classmethod
+    def __deprecated_get_value_for_org(cls, org, val_name, default=None):
+        """
+        This method iterates over all the TenantConfig and Microsite objects, set the cache for the given
+        val_name and all organizations and returns the value for the given org.
+
+        DEPRECATED: use __get_value_for_org method instead.
```

Comment:
could you please leave a comment here stating why the deprecated method exists at all still. In my understanding we will never remove it since the new method requires mysql which we not always have. E.g the testing environment.